packages/toolkit/src/view/model/ModelsTable.tsx

```diff
@@ -76,9 +76,9 @@ export const ModelsTable = (props: ModelsTableProps) => {
           <div className="grid justify-items-center">
             <StateLabel
               state={
-                modelsWatchState.isSuccess
-                  ? modelsWatchState.data[name]?.state
-                  : "STATE_UNSPECIFIED"
+                (modelsWatchState.isSuccess &&
+                  modelsWatchState.data[name]?.state) ||
+                "STATE_UNSPECIFIED"
```

Comment:
@orangecoloured Please use `ternary operator` instead of short-cutting in this scenario (We only use short-cutting when we want to set default value)